test/unit-tests/MultiBridgeMessageReceiver.t.sol

```diff
@@ -462,13 +470,86 @@ contract MultiBridgeMessageReceiverTest is Setup {
         uint64 newQuorum = 1;
 
         /// @dev removes the newly updated adapter by reducing quorum by one
-        receiver.updateQuorumAndReceiverAdapter(newQuorum, adapters, new bool[](1));
+        receiver.updateReceiverAdaptersAndQuorum(adapters, new bool[](1), newQuorum);
 
         /// @dev asserts the quorum and adapter lengths
         assertEq(receiver.quorum(), newQuorum);
         assertEq(receiver.isTrustedExecutor(adapters[0]), false);
     }
 
```

Comment:
given we have a bit more logic in the constructor now, can we add a `test_constructor` that verifies the default setup

Reply:
Done.